src/Data/Uniques/amulet.lua

```diff
@@ -613,6 +613,18 @@ Implicits: 1
 {tags:speed}30% increased Projectile Speed
 {tags:speed}10% increased Movement Speed
 ]],[[
+Leadership's Price
+Onyx Amulet
+Requires Level 68
+Implicits: 1
+{tags:jewellery_attribute}+(10–16) to all Attributes
++(-3–3)% to maximum Fire Resistance
++(-3–3)% to maximum Cold Resistance
++(-3–3)% to maximum Lightning Resistance
```

Comment:
Technically these should have `jewellery_resistance` as a tag, but 3% * 20% rounds down to zero anyway